docs/source/user/radius_monitoring.rst

```diff
@@ -0,0 +1,81 @@
+Integration with OpenWISP Monitoring
+------------------------------------
+
+OpenWISP RADIUS includes an optional Django sub-app that adds integration with
+`OpenWISP Monitoring <https://github.com/openwisp/openwisp-monitoring#openwisp-monitoring>`_
+to provide RADIUS metrics.
+
+.. image:: /images/radius-dashboard-charts.png
+   :alt: RADIUS session dashboard charts
+
+RADIUS metrics
+==============
+
+1. User registrations
+~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: /images/user-registration-chart.png
+   :alt: User registration chart
+
+This chart shows number of users signed up using different registration methods.
+
+2. Total user registrations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
this line is longer than the title, let's also update the charts images once we are done with everything else